tasks.py

```diff
@@ -233,7 +233,7 @@ def dev_env(ctx, architecture="amd64", name="kind", cni=None, protocol=None, nod
         if cni:
             networking_config["disableDefaultCNI"] = True
         if ip_family != "ipv4":
-            networking_config["disableDefaultCNI"] = ip_family
+            networking_config["ipFamily"] = ip_family
```

Comment:
ah ok. This confused me in the previous commit. Not sure if you intend to squash commits but could you add this change to the first commit.